### ISSUE TITLE
[Bug] Fix the contrast issue(s)

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/flickpicker#79]
The current contrast ratio using the browser WAVE tool is 2.07:1 this fails the WCAG requirements for AA and AAA. 

the foreground color is #0D6EFD 
the background color is #5B2E91

Fix is so that the contrast meets the requirements but stays within the color range we are using.

**Comment by gbowne1** [comment]
I think we should try changing #0D6EFD to:

#1A73E8
#084298

Then maybe white #ffffff text on purple backgrounds
